internal/credit/creditdriver/grant.go

```diff
@@ -249,3 +293,33 @@ func (b *builder) GetLedgerGrant() GetLedgerGrantHandler {
 		)...,
 	)
 }
+
+func mapGrantToAPI(grant credit.Grant) api.LedgerGrantResponse {
+	var featureID string
+
+	if grant.FeatureID != nil {
```

Comment:
Instead of these please use something like this: https://github.com/openmeterio/openmeter/blob/fcce3d310c161bd8ab17bfe0bbe88b933503dd58/pkg/defaultx/default.go#L3

Reply:
changed everywhere